src/data/zwave_js.ts

```diff
@@ -268,6 +307,43 @@ export const validateDskAndEnterPin = (
     pin,
   });
 
+export const supportsFeature = (
```

Comment:
Can we prefix these functions with `zwave`? Auto-import VS Code suggestions are otherwise going to confuse us.